test/langtools/tools/javac/annotations/typeAnnotations/CompletionErrorOnEnclosingType.java

```diff
@@ -80,7 +80,7 @@ class C {
                 new JavacTask(tb)
                         .outdir(out)
                         .classpath(out)
-                        .options("-XDrawDiagnostics")
+                        .options("-XDrawDiagnostics", "-XDaddTypeAnnotationsToSymbol=true")
```

Comment:
I wonder if this new option would need a CSR.

Reply:
javac `-XD` flags are internal/undocumented/unsupported, and as far as I know generally don't have CSRs. @vicente-romero-oracle does that sound right?

Reply:
Yeah, came to the same conclusion. Thanks.